src/inverseRobot.js

```diff
@@ -1,13 +1,20 @@
 'use strict';
 
-/*
- * @param {object} robot
- *
- * @return {object}
-*/
-
 function inverseRobot(robot) {
-  // write code here
+  const correctObj = {};
+  let value;
```

Comment:
Not sure, you need this variable in closure, it make sense only if you need it outside your loop